sim/shaman/bloodlust.go

```diff
@@ -12,7 +12,7 @@ func (shaman *Shaman) BloodlustActionID() core.ActionID {
 }
 
 func (shaman *Shaman) registerBloodlustCD() {
-	if !shaman.SelfBuffs.Bloodlust {
+	if !shaman.SelfBuffs.Bloodlust && !shaman.IsUsingAPL {
```

Comment:
we want the shaman lust spell to be always registered as an option if using the APL (even if the settings UI has disabled bloodlust)